packages/loader/index.js

```diff
@@ -5,7 +5,12 @@ module.exports = async function(content) {
   const callback = this.async()
   const options = getOptions(this)
 
-  const result = await mdx(content, options || {})
+  const filename = this.resourcePath;
```

Comment:
For the name, I’d personally suggest `options.filePath`, as you’re setting `file.path`, but it doesn’t really matter!

Re the style, maybe use this, looks cleaner to me:

```js
options = Object.assign({}, getOptions(this), {filename: this.resourcePath})
```

Also: is there a specific cwd somehow set by webpack? we could use that for [file.cwd](https://github.com/vfile/vfile#vfilecwd)

Reply:
Ohh that is much cleaner! The reason that I put it in an `if` was that when I tested it with `filename: undefined` it hanged for some reason. I can test a bit more, maybe it was something else.

I also like `filepath` much more, I will change it!